server/src/config/dbBuild.js

```diff
@@ -11,20 +11,27 @@ const { sequelize } = require('./connection');
 const data = require('./data.json');
 
 const importDataToDatabase = async () => {
-  await sequelize.sync({ force: true });
+  try {
+    await sequelize.sync({ force: true });
 
-  await Promise.all([
-    ...data.users.map((item) => User.create(item)),
-    ...data.categories.map((item) => Category.create(item)),
-  ]);
-  await Promise.all([
-    ...data.customers.map((item) => Customer.create(item)),
-    ...data.restaurants.map((item) => Restaurant.create(item)),
-  ]);
-  await Promise.all(data.meals.map((item) => Meal.create(item)));
-  await Promise.all(data.discountCode.map((item) => DiscountCode.create(item)));
+    await Promise.all([
+      data.users.map((item) => User.create(item)),
+      data.categories.map((item) => Category.create(item)),
+    ]);
+    await Promise.all([
+      data.customers.map((item) => Customer.create(item)),
+      data.restaurants.map((item) => Restaurant.create(item)),
+    ]);
+    await Promise.all(data.meals.map((item) => Meal.create(item)));
+    await Promise.all(
+      data.discountCode.map((item) => DiscountCode.create(item))
+    );
 
-  console.log('Database filled with data!');
+    console.log('Database filled with data!');
+  } catch (err) {
+    console.log(err);
+    console.log('hhhhhhhhhhhhhhhhhhhhhhhhhhh');
```

Comment:
whats happening here

Reply:
I was debuging and forgot to remove it 🤣